src/main/java/lesson20/Demo.java

```diff
@@ -0,0 +1,45 @@
+package lesson20;
+
+import lesson20.AdapterExample.SquerePieToPizza;
+import lesson20.AdapterExample.SquerePie;
+import lesson20.Factory.AbstractFactory;
+import lesson20.Factory.FactoryProducer;
+import lesson20.ProxyPizza.ProxyPeperony;
+import lesson20.Wrapper.Delivery;
+import lesson20.Wrapper.HotDelivery;
+
+public class Demo {
+    public static void main(String[] args) {
+        Delivery delivery = new Delivery();
```

Comment:
You use concretic factory|class. It is hardcode. Read https://www.baeldung.com/java-abstract-factory-pattern

Reply:
Ok, I saw below =)